docs/docs/guides/platform/weave-self-managed.md

```diff
@@ -66,12 +66,6 @@ Modify the following parameters:
 W&B recommends keeping the `clusterName` value in `values.yaml` set to `weave_cluster`.  This is the expected cluster name when W&B Weave runs the database migration. If you need to use a different name, see the [Setting `clusterName`](#setting-clustername) section for more information.
 
 ```yaml
-# Stable version
-image:
-  registry: docker.io
-  repository: bitnami/clickhouse
-  tag: 24.8
```

Comment:
Where did specifying the clickhouse image go to?

Reply:
I tried to test using the stable version, but it seems the Bitnami image has something else that prevents the stable version from working correctly.
That's the reason I removed it.
The image is set in the chart version `8.0.10` is this:

```
image:
  registry: docker.io
  repository: bitnami/clickhouse
  tag: 25.3.2-debian-12-r3  
```